protected/views/site/index.php

```diff
@@ -17,9 +17,9 @@
                 <div class="row">
                     <div class="col-xs-8">
                         <div class="underline-title">
-                            <ol class="breadcrumb pull-right" style="cursor:pointer">
-                                <li><span>+ more</span></li>
-                            </ol>
+                            <div class="breadcrumb pull-right" style="cursor:pointer">
+                                <span class="breadcrumb-item" data-toggle="datasettypes">+ more</span>
```

Comment:
added additional data attribute as it made more sense to me to decouple css classes from on click handlers